h/buildext.py

```diff
@@ -114,6 +75,14 @@ def copytree(src, dst):
             shutil.copyfile(s, d)
 
 
+def embed_code(request):
+    if request.webassets_env.url.startswith('chrome-extension:'):
```

Comment:
Are you saying I should reverse the condition?
On Mar 5, 2015 11:58 AM, "Kristof Csillag" <notifications@github.com> wrote:

> In h/buildext.py
> <https://github.com/hypothesis/h/pull/1996#discussion_r25896154>:
>
> > @@ -114,6 +75,14 @@ def copytree(src, dst):
> >              shutil.copyfile(s, d)
> >
> >
> > +def embed_code(request):
> > +    if request.webassets_env.url.startswith('chrome-extension:'):
>
> Isn't that limitation specific to FF? 2015.03.05. 20:57 ezt írta ("Randall
> Leeds" <notifications@github.com>):
> … <#14beb84a5eb53d66_>
>  In h/buildext.py <
> https://github.com/hypothesis/h/pull/1996#discussion_r25896031>: > @@
> -114,6 +75,14 @@ def copytree(src, dst): > shutil.copyfile(s, d) > > > +def
> embed_code(request): > + if
> request.webassets_env.url.startswith('chrome-extension:'): No. Loading
> iframes from resource: doesn't work. … <#14beb83536e78190_> On Mar 5, 2015
> 4:07 AM, "Nick Stenning" <notifications@github.com> wrote: In
> h/buildext.py <
> https://github.com/hypothesis/h/pull/1996#discussion_r25858600>: > @@
> -114,6 +75,14 @@ def copytree(src, dst): > shutil.copyfile(s, d) > > > +def
> embed_code(request): > + if
> request.webassets_env.url.startswith('chrome-extension:'): Does this not
> need to be done for Firefox too? — Reply to this email directly or view it
> on GitHub <https://github.com/hypothesis/h/pull/1996/files#r25858600>. —
> Reply to this email directly or view it on GitHub <
> https://github.com/hypothesis/h/pull/1996/files#r25896031>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hypothesis/h/pull/1996/files#r25896154>.
>